_examples/storage/README.md

```diff
@@ -0,0 +1,106 @@
+
+```
+docker run -d -p 3000:3000 --name aerospike aerospike/aerospike-server
+```
+
+Now, we need the sample code.
+
+```
+go get -u github.com/mcuadros/go-git-aerospike/...
+```
+
+Running this command the binary `go-git-aerospike` will become available, if you have `GOPATH` on your `PATH`, you are ready to go. If not, this could be a great moment to add it.
+
+Usage
+-----
+
+### Cloning the repository into the database
+
+Running the command `go-git-aerospike` with the `clone` option followed by the URL of a git repository like:
+
+```sh
+go-git-aerospike clone https://github.com/src-d/flamingo.git
+```
+
+The repository is stored on the aerospike database. This means that all the internal objects like commits, trees, blobs and tags are `records` in differents `sets` in the `test` namespace:
+
+```sql
+aql> SELECT hash, type, url FROM test.commit
+```
+
+```
++--------------------------------------------+----------+-------+-----------------------------------+
+| hash                                       | type     | blob  | url                               |
++--------------------------------------------+----------+-------+-----------------------------------+
+| "c94450c805876e49b38d2ff1103b8c09cdd2aef4" | "commit" | 00 00 | ...github.com/src-d/flamingo.git" |
+| "7f71640877608ee9cfe584fac216f03f9aebb523" | "commit" | 00 00 | ...github.com/src-d/flamingo.git" |
+| "255f097450dd91812c4eb7b9e0d3a4f034f2acaf" | "commit" | 00 00 | ...github.com/src-d/flamingo.git" |
++--------------------------------------------+----------+-------+-----------------------------------+
+102 rows in set (0.071 secs)
+```
+
+And also the references and the configuration (remotes) are stored on it.
+
+```sql
+aql> SELECT name, target, url FROM test.reference
+```
+```
++------------------------------+--------------------------------------------+-----------------------+
+| name                         | target                                     | url                   |
++------------------------------+--------------------------------------------+-----------------------+
+| "HEAD"                       | "ref: refs/heads/master"                   | ...rc-d/flamingo.git" |
+| "refs/heads/master"          | "ed3e1aa2e46584cb803ed356cb5d8855f6d05660" | ...rc-d/flamingo.git" |
+| "refs/remotes/origin/master" | "ed3e1aa2e46584cb803ed356cb5d8855f6d05660" | ...rc-d/flamingo.git" |
++------------------------------+--------------------------------------------+-----------------------+
+3 rows in set (0.046 secs)
+```
+
+### Reading the repository
+
+Running the `log` command:
+
+```sh
+go-git-aerospike log https://github.com/src-d/flamingo.git
+```
+
+The URL of the repository is the way that the objects in the `sets` are discriminated, since we can clone several repository to the same database.
+
+we get something similar to the output of the command `git log --oneline`, over a classic `git` repository:
```

Comment:
I would remove the phase starting in `, over a classic`. People might wonder if it is not a classic repo, which are the differences?